packages/contracts/test/voting/token-voting.ts

```diff
@@ -1104,8 +1114,11 @@ describe('TokenVoting', function () {
       });
 
       it('executes early if participation and support are met and the vote outcome cannot change anymore', async () => {
-        await governanceErc20Mock.mock.getPastVotes.returns(50);
         await voting.connect(signers[0]).vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[1]).vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[2]).vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[3]).vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[4]).vote(id, VoteOption.Yes, false);
```

Comment:
could also be optimized with a loop and `Promise.all`

Reply:
Improved in bbc468ded7f531d27ce685c55d3dd166acc20bef